overrides.rb

```diff
@@ -93,6 +93,8 @@
             pkg.define 'ROCK_TEST_LOG_DIR', pkg.test_utility.source_dir
         end
         pkg.define "CMAKE_EXPORT_COMPILE_COMMANDS", "ON"
+        pkg.define 'ROCK_TEST_BOOST_FORMAT',
+                   pkg.ws.config.get('rock_test_boost_format', 'XML')
```

Comment:
I think this should be defined only when tests are enabled... CMake will generate warnings for unused definitions

Reply:
Done